hamcrest/src/main/java/org/hamcrest/text/IsBlankString.java

```diff
@@ -20,7 +20,16 @@ private IsBlankString() { }
 
     @Override
     public boolean matchesSafely(String item) {
-        return item.isBlank();
+        final int length = item.length();
+        int offset = 0;
+        while(offset < length) {
+            final int codePoint = item.codePointAt(offset);
+            if(!Character.isWhitespace(codePoint)) {
```

Comment:
Same as above: consider sticking a space between `if` and the opening parenthesis.